### DISCUSSION TITLE
How to implement subscription type of payment?

### DISCUSSION THREAD

**Discussion by ahsi1999** [flutter-stripe/flutter_stripe#1501] [Q&A]
Using payment intent I am able to do one time payment of stripe but If I want to implement subscription type of payment then how am I supposed to do it?

**Comment by remonh87** [comment]
I would recommend looking into the stripe documentation: https://stripe.com/docs/billing/subscriptions/overview

Normally you collect payment data through an setupintent and create subscriptions after it. See also https://stackoverflow.com/questions/62454487/stripe-v3-setupintents-and-subscriptions

**Reply by ahsi1999** [reply]
I tried it but couldn't implement it successfully. Can you please provide me a sample code? Because it works on test but does not works on live so don't know what to do?
